src/app/base/components/AppSideNavigation/AppSideNavItems/AppSideNavItems.tsx

```diff
@@ -18,6 +18,7 @@ type Props = {
   isAuthenticated: boolean;
   logout: () => void;
   path: string;
+  setIsCollapsed: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
I'd suggest to reuse `SideNavigationProps` instead of defining it again, this will make it easier to maintain.